bin/all_sky_search/pycbc_fit_sngls_split_binned

```diff
@@ -136,28 +137,21 @@ for ex_p in extparams:
             ]
```

Comment:
Consider rewriting this as a simple for loop rather than list comprehension? 
```
durations = []
for ref in trigf[args.ifo + '/template_duration_template'][:]:
    dur.append(trigf[args.ifo + '/template_duration'][ref][0])
params['template_duration'] = np.array(durations)
```
- this is 4 lines. 
Also, I don't understand what the comment about the quirk of h5py implies here - the code doesn't seem to do anything special for the case with no triggers